clinica/pipelines/deeplearning_prepare_data/README.md

```diff
@@ -0,0 +1,73 @@
+# `deeplearning-prepare-data`` - <VERY_SHORT_DESCRIPTION>
+
+<SHORT_DESCRIPTION>
+
+
+## Dependencies
+<!-- If you installed the docker image of Clinica, nothing is required. -->
+
+If you only installed the core of Clinica, this pipeline needs the installation
+of **<software_package>** on your computer. You can find how to install this
+software on the [third-party](../../Third-party) page.
+
+
+## Running the pipeline
```

Comment:
File to delete

Reply:
Done!
I wonder why this file is generated by the jinja2 template?
(Maybe useful for other pipelines, but I don't have enough experience with Clinica to confirm that).

Reply:
At the beginning, this was generated to enforce harmonised Wiki description and it was planned (but abandoned) that Wiki would be also on the repo.

Maybe we can remove this part from `clinica generate`? 🤔